spec/system/admin/invoice_print_spec.rb

```diff
@@ -254,12 +254,19 @@
           expect(page).to have_content "Admin & Handling $120.00"
           # Shipping
           expect(page).to have_content "Shipping $100.55 10.0%"
+          # Tax totals
+          expect(page).to have_content "Total tax (10.0%): $9.14 " \
+                                       "Total tax (15.0%): $15.65 Total tax (20.0%): $250.08"
           # Order Totals
           expect(page).to have_content "Total (Incl. tax): $1,733.54"
           expect(page).to have_content "Total tax (15.0%): $15.65"
           expect(page).to have_content "Total tax (10.0%): $9.14"
           expect(page).to have_content "Total tax (20.0%): $250.08"
           expect(page).to have_content "Total (Excl. tax): $1,458.67"
+
+          # Tax totals
+          expect(page).to have_content "Total tax (10.0%): $9.14 " \
+                                       "Total tax (15.0%): $15.65 Total tax (20.0%): $250.08"
```

Comment:
Why are we checking three times for the same content?

Reply:
This went unnoticed.
I'll need to find out the best to find a way to preview the changes in each commit, before actually pushing, to prevent these mistakes.

Reply:
Not sure which tools you use. I'm usually on the terminal. Before I commit all changes I usually do a diff:
```
git diff HEAD
git commit -a
```

Or if I want to check all commits on my branch:

```
git show master..
```

Reply:
I'm on terminal too, will give that a try - thank you.